src/store/localization/reducers.tsx

```diff
@@ -20,6 +20,17 @@ const initialState = {
 			"withInternet": "With an Internet connection",
 			"withoutInternet": "Without an Internet connection",
 			"mustBeOnline": "You must be connected to the Internet to Log In!",
+			"onlineAdmin": "Online Admin",
+			"onlineAdminDesc": "Project Admin is online.",
+			"offlineAdmin": "Always Offline",
+			"offlineAdminDesc": "Project Admin is offline.",
+			"onlineWorkDesc": "Transcription can be done online or offline.",
+			"offlineWorkDesc": "All work is always done offline.",
+			"back": "Back",
+			"chooseAnother": "To choose another user, Log out...",
+			"noOnlineUsers1": "To allow offline use, while online mark your project as 'Available Offline' in your project settings.",
+			"or": "or",
+			"noOnlineUsers2": "Import a Project that your admin has exported.",
```

Comment:
I think we can put this in one literal with hard line breaks? The concern is that when translated to other languages, it is very tricky to translate parts of sentences. Some languages may represent a either/or sentence with more than just the word OR in the middle.